agent/dns.go

```diff
@@ -135,6 +135,37 @@ func (d *DNSServer) ListenAndServe(network, addr string, notif func()) error {
 	return d.Server.ListenAndServe()
 }
 
+func setEDNS(request *dns.Msg, response *dns.Msg, ecsGlobal bool) {
```

Comment:
I would add a func comment here. I realize its not exported but even then I try to do it unless the implementation is trivial.